src/powfacpy/base_interface.py

```diff
@@ -833,7 +833,34 @@ def get_loc_name_with_class(powerfactory_objects):
     if not is_list:
       loc_name_with_class = loc_name_with_class[0]  
     return loc_name_with_class
+    
+  def create_comtrade_obj(self,file_path: str,parent_folder=None):
```

Comment:
add spaces between comma and next argument like this:
`def create_comtrade_obj(self, file_path: str, parent_folder=None):`

(for all function definitions, but only commented here)

Reply:
Ok, I did this for the new file script_interface.py.
In a future commit, this should be fixed for the whole package.